### PR TITLE
[bitnami/contour] Added option to set ingress-status-address.

### PR DESCRIPTION
**Pull request by kahirokunn**
### Description of the change

If you do not use the LoadBalancer Service, you may want to fill in the ingress-status-address yourself.
For example, you may want to fill it with the status-address of ALB Ingress.
I have added a flag to realize this.

https://github.com/projectcontour/contour/issues/4951

### Benefits

Even if you do not use the LoadBalancer Service, you can fill in the status of HTTPProxy.
Integration with ExternalDNS and other services will be more seamless.

### Possible drawbacks

None

### Applicable issues

  - fixes #

### Additional information

Since it conflicts with the following, rebase it once after the following is merged.
https://github.com/bitnami/charts/pull/14512

### Checklist

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/). This is *not necessary* when the changes only affect README.md files.
- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [x] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [x] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/main/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/contour/Chart.yaml [hunk 1]
```diff
@@ -25,4 +25,4 @@ sources:
   - https://github.com/envoyproxy/envoy
   - https://github.com/bitnami/containers/tree/main/bitnami/contour
   - https://projectcontour.io
-version: 10.1.5
+version: 10.3.0
```
Comment (Mauraza) [inline bitnami/contour/Chart.yaml:28]:
```suggestion
version: 10.2.0
```

**Review by Mauraza — COMMENTED** [review]
Hi @kahirokunn,

Thanks for the contribution! Could you check the small change?

File: bitnami/contour/Chart.yaml (hunk 1 above)
Comment (kahirokunn) [reply bitnami/contour/Chart.yaml]: Thx! I fixed.

**Review by Mauraza — APPROVED** [review]
Hi @kahirokunn,

Thanks again for the contribution!! LGTM!

**Pull request merged**